apps/marketing/content/blog/public-api.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: 'Building the Documenso Public API - The Why and How'
+description: 'This article talks about the need for the public API and the process of building it. It also discusses the requirements we had to meet and the constraints we had to work within.'
+authorName: 'Catalin'
+authorImage: '/blog/blog-author-catalin.webp'
+authorRole: 'Developer'
+date: 2024-03-08
+tags:
+  - Development
+  - API
+---
+
+This article covers the process of building the public API for Documenso. It starts by explaining why the API was needed for a digital document signing company in the first place. Then, it'll dive into the steps we took to build it. Lastly, it'll present the requirements we had to meet and the constraints we had to work within.
+
+## Why the public API
+
+We decided to build the public API to open a new way of interacting with Documenso. While the web app does the job well, there are use cases where it's not enough. In those cases, the users might want to interact with the platform programmatically. Usually, that's for integrating Documenso with other applications.
+
+The new public API enables them to do that. The users can integrate Documenso's functionalities within other applications to automate tasks, create custom solutions, and build custom workflows, to name just a few.
+
+The API provides 12 endpoints at the time of writing this article:
```

Comment:
and also Instead of the users, whats your thought on putting it as `You can integrate Documenso's functionalities`

Reply:
I don't get it. Where should I make the changes and what changes? 😆

Reply:
It was mostly from Lines 18-21, as we are putting this on our company blog, and we writing it as us (when referring to documenso) and you(when referring to the users) would be much more pleasant on the eyes is what I feel 😶